### PR TITLE
feat(swcx): remove swcx from @swc/core pkgs

### PR DESCRIPTION
**Pull request by kwonoj**
**Description:**

Part of https://github.com/swc-project/swc/discussions/6479. Now `@swc/cli` exposes `swcx` command for the new cli binary, we do not need to duplicate binaries in `@swc/core`.

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

**Comment by kdy1** [comment]
I'm not sure if this will work. Can you use `ci` branch to verify publish action?

**Comment by kwonoj** [comment]
This is mostly revert of https://github.com/swc-project/swc/pull/3784. Any particular place you are concerned?

File: scripts/cli_artifacts.sh [hunk 1]
```diff
@@ -16,9 +16,7 @@ do
   if [ -f "$CLI_BINARY_PATH" ]; then
       chmod +x $CLI_BINARY_PATH
       cp -v $CLI_BINARY_PATH ./artifacts_cli/swc-$BINDING_ABI
-      mv $CLI_BINARY_PATH ./scripts/npm/$BINDING_ABI
```
Comment (kdy1) [inline scripts/cli_artifacts.sh:19]: I think this can break the CI pipeline

Comment (kwonoj) [reply scripts/cli_artifacts.sh:19]: How? This is moving swcx binary to npm pkg to be published, not the actual .node bindings.

Comment (kwonoj) [reply scripts/cli_artifacts.sh:19]: I can verify and report back.

Comment (kwonoj) [reply scripts/cli_artifacts.sh:19]: https://github.com/kwonoj/swc/actions/runs/3798606577/jobs/6460886033#step:9:110 CI passes publish (except real publish) with artifacts.

Comment (kdy1) [reply scripts/cli_artifacts.sh:19]: Thank you!

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - dbg-swc

**Review by kdy1 — APPROVED** [review]

**Pull request merged**